### PR TITLE
[Bugfix] Fix dropdown

### PR DESCRIPTION
**Pull request by damassi**
MVC UI code is terrifying. Fixes issue with dropdown not updating: 

sort

### CODE REVIEW

File: src/desktop/components/artwork_filter_2/views/filters_view.coffee [hunk 1]
```diff
@@ -78,6 +78,7 @@ module.exports = class ArtworkFiltersView extends Backbone.View
     e.preventDefault()
     @resetPagination()
     key = ($el = $(e.target)).data('key')
+    @params.set key, null # FIXME: Force an update to the model and then unset key
```
Comment (mzikherman) [inline src/desktop/components/artwork_filter_2/views/filters_view.coffee]: `@params.unset 'key'`?

Comment (damassi) [reply src/desktop/components/artwork_filter_2/views/filters_view.coffee]: For whatever reason when using `unset key` it seems to trigger an event update before its actually unset, so this explicitely forces it to a value (to trigger an update) and the unsets it. Not sure, but can return to this at some point

Comment (mzikherman) [reply src/desktop/components/artwork_filter_2/views/filters_view.coffee]: oh lol that's the next line. Shouldn't `unset` fire a `change` event (and then presumably there is some listener to `@params.on 'change:page'` or something)?

Comment (damassi) [reply src/desktop/components/artwork_filter_2/views/filters_view.coffee]: Yup, it listens to things [here](https://github.com/damassi/force/blob/more-pagination-qa/src/desktop/components/artwork_filter_2/models/filter.coffee#L22), yet by the time we get to [here](https://github.com/damassi/force/blob/more-pagination-qa/src/desktop/components/artwork_filter_2/models/filter.coffee#L42) it still thinks the filter param is set on the model. By setting it to `null` first it strips it from the query to MP, and then cleans up with an `unset`.

Comment (mzikherman) [reply src/desktop/components/artwork_filter_2/views/filters_view.coffee]: Yea, basically was just commenting (w/o having looked at this code before), in that it seems like _two_ updates get triggered: once from this `set`, and once from the `unset` (ie- none have `silent: true`). So was wondering if maybe something was out of whack there, but yea all this backbone mvc stuff.... 😿

**Pull request merged**